.github/workflows/test-coverage.yml

```
@@ -0,0 +1,13 @@
+name: 'coverage'
+on: push
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    steps:
+      - uses: actions/checkout@v1
+      - uses: artiomtr/jest-coverage-report-action@v2.0-rc.5
```

Regarding [the convention](https://nimblehq.co/compass/development/code-conventions/github-actions/), should we add some spaces here? 

```suggestion
name: 'coverage'

on: push

jobs:
  coverage:
    runs-on: ubuntu-latest
    if: "!contains(github.event.head_commit.message, '[skip ci]')"
    steps:
      - uses: actions/checkout@v1

      - uses: artiomtr/jest-coverage-report-action@v2.0-rc.5
```

---

✅ 👌 6c62991